__tests__/components/CrossData.test.js

```diff
@@ -0,0 +1,17 @@
+import React, { useContext } from 'react';
+import renderer from 'react-test-renderer';
+
+import { CrossData } from '../../src/components';
+
+describe('CrossData', () => {
```

Comment:
there is this test but no `snap` file with a snapshot. if this test should not be considered or not run, it could be marked with `skip` https://codewithhugo.com/run-skip-single-jest-test/

Reply:
I'm not running all the tests together, just the individual test files - but thanks for reminding me of .skip will help me soon!